integration/spark/src/main/scala/org/apache/carbondata/events/Events.scala

```diff
@@ -195,7 +195,7 @@ trait CreateCarbonRelationEventInfo {
  */
 trait MVEventsInfo {
   val sparkSession: SparkSession
-  val storePath: String
+  val systemDirectoryPath: String
```

Comment:
The field name changed, but i can not found any code which used the field has be changed, it's will be OK？

Reply:
yes, this is just the event, if any user wants to implement a listener for this event, then he will use these variable. So it is ok